### PR TITLE
feat(samples): updated execution order, hello world, transformers and retry to not use Container.Op. Part of #5687

### PR DESCRIPTION
**Pull request by NikeNano**
**Description of your changes:**

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

**Comment by NikeNano** [comment]
@chensun maybe you could take a look on this?

**Comment by chensun** [comment]
Part of https://github.com/kubeflow/pipelines/issues/5801

File: samples/core/pipeline_transformers/pipeline_transformers.py [hunk 1]
```diff
@@ -16,27 +16,28 @@
 
 import kfp
 from kfp import dsl
+import kfp.components as comp
 
+
+@comp.create_component_from_func
 def print_op(msg):
```
[the remark this answers is not in the document]
Comment (NikeNano) [reply samples/core/pipeline_transformers/pipeline_transformers.py:23]: Ok, so Vertex AI is the same as V2? Will update.

**Comment by NikeNano** [comment]
I dont think the failed tests are related to this PR, could you take a look again @chensun? Thanks!

**Comment by chensun** [comment]
/lgtm
/approve

Thank you!

**Pull request merged**